raft-ann-bench/gpu/Dockerfile

```diff
@@ -14,24 +14,24 @@ COPY condarc /opt/conda/.condarc
 
 # Create a data folder accessible by any user so mounted volumes under it can be accessible
 # when the user passes their uid to docker run with -u $(id -u)
-RUN mkdir /data && chmod 777 /data
+# Also add the conda_prefix config to the global bashrc file so that all users have it correctly configured.
+RUN mkdir /data && chmod 777 /data \
+    &&  echo "export CONDA_PREFIX=/opt/conda" >> /etc/bash.bashrc
```

Comment:
I don't think this is needed because `rapidsai/mambaforge-cuda` will automatically activate the `base` env.

Reply:
I thought so at first but it does for already created users, lik the rapids user environment, it doesn't for the user passed by uid as we found out during testing, which makes this necessary.